ksql-rest-app/src/test/resources/ksql-plan-schema/schema.json

```diff
@@ -0,0 +1,1062 @@
+          "enum" : [ "createTableV1" ],
+          "default" : "createTableV1"
+        },
+        "sqlExpression" : {
+          "type" : "string"
+        },
+        "sourceName" : {
+          "type" : "string"
+        },
+        "schema" : {
+          "type" : "string"
+        },
+        "keyField" : {
+          "type" : "string"
+        },
+        "timestampExtractionPolicy" : {
+          "oneOf" : [ {
+            "$ref" : "#/definitions/MetadataTimestampExtractionPolicy"
+          }, {
+            "$ref" : "#/definitions/StringTimestampExtractionPolicy"
+          }, {
+            "$ref" : "#/definitions/LongColumnTimestampExtractionPolicy"
+          } ]
+        },
+        "serdeOptions" : {
+          "type" : "array",
+          "items" : {
+            "type" : "string",
+            "enum" : [ "UNWRAP_SINGLE_VALUES" ]
+          }
+        },
+        "topic" : {
+          "$ref" : "#/definitions/KsqlTopic"
+        }
+      },
+      "title" : "createTableV1",
+      "required" : [ "@type", "sqlExpression", "sourceName", "schema", "keyField", "timestampExtractionPolicy", "serdeOptions", "topic" ]
+    },
+    "RegisterTypeCommand" : {
+      "type" : "object",
+      "additionalProperties" : false,
+      "properties" : {
+        "@type" : {
+          "type" : "string",
+          "enum" : [ "registerTypeV1" ],
+          "default" : "registerTypeV1"
+        },
+        "type" : {
+          "type" : "string"
+        },
+        "name" : {
+          "type" : "string"
+        }
+      },
+      "title" : "registerTypeV1",
+      "required" : [ "@type", "type", "name" ]
+    },
+    "DropSourceCommand" : {
+      "type" : "object",
+      "additionalProperties" : false,
+      "properties" : {
+        "@type" : {
+          "type" : "string",
+          "enum" : [ "dropSourceV1" ],
+          "default" : "dropSourceV1"
+        },
+        "sourceName" : {
+          "type" : "string"
+        }
+      },
+      "title" : "dropSourceV1",
+      "required" : [ "@type", "sourceName" ]
+    },
+    "DropTypeCommand" : {
+      "type" : "object",
+      "additionalProperties" : false,
+      "properties" : {
+        "@type" : {
+          "type" : "string",
+          "enum" : [ "dropTypeV1" ],
+          "default" : "dropTypeV1"
+        },
+        "typeName" : {
+          "type" : "string"
+        }
+      },
+      "title" : "dropTypeV1",
+      "required" : [ "@type", "typeName" ]
+    },
+    "QueryPlan" : {
+      "type" : "object",
+      "additionalProperties" : false,
+      "properties" : {
+        "sources" : {
+          "type" : "array",
+          "items" : {
+            "type" : "string"
+          }
+        },
+        "sink" : {
+          "type" : "string"
+        },
+        "physicalPlan" : {
+          "$ref" : "#/definitions/PhysicalPlan[Object]"
+        }
+      },
+      "required" : [ "sources", "sink", "physicalPlan" ]
+    },
+    "PhysicalPlan[Object]" : {
```

Comment:
We should remove the generics on PhysicalPlan so that we drop the `[Object]` here.  The generics aren't actually of any use.